src/stream/src/executor/debug/update_check.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Debug;
+
+use async_trait::async_trait;
+use itertools::Itertools;
+use risingwave_common::array::Op;
+use risingwave_common::error::Result;
+
+use crate::executor::{Executor, Message};
+
+/// [`UpdateCheckExecutor`] checks whether the two rows of updates are next to each other.
+#[derive(Debug)]
+pub struct UpdateCheckExecutor {
+    /// The input of the current executor.
+    input: Box<dyn Executor>,
+}
+
+impl UpdateCheckExecutor {
+    pub fn new(input: Box<dyn Executor>) -> Self {
+        Self { input }
+    }
+}
+
+#[async_trait]
+impl super::DebugExecutor for UpdateCheckExecutor {
```

Comment:
Can we merge all debug executors into one? The backtrace would be disaster...

Reply:
Next time for sure!